solr/core/src/java/org/apache/solr/cloud/Overseer.java

```diff
@@ -173,6 +175,9 @@
 
     private final Stats zkStats;
 
+    private final SolrMetricManager metricManager = new SolrMetricManager();
```

Comment:
This is wrong - you should never create instances of SolrMetricManager in your components. Always use the single instance constructed in `CoreContainer.getMetricManager()`.